Data Section Update (#1127)
ACCESS-Hive/access-hive.github.io: docs/model_evaluation/data/finding/interactive_catalogue.md

!!! warning
    The ACCESS-NRI Interactive Data Catalogue is currently under development - functionality may therefore change from that described here.

>     For the most reliable access to ACCESS model data, please use the ACCESS-NRI Intake Catalogue via Python as described in the [ACCESS-NRI Intake Catalogue documentation](https://access-nri-intake-catalog.readthedocs.io/en/latest/index.html).

Reviewer: ```suggestion
!!! release
    The ACCESS-NRI Interactive Data Catalogue is currently under development, and its functionality may differ from what is described in this page.
    For reliably retrieving ACCESS model data, please use the [ACCESS-NRI Intake Catalogue Python API](https://access-nri-intake-catalog.readthedocs.io/en/latest/index.html).
```

> # What is the ACCESS-NRI Interactive Data Catalog?

Reviewer: ```suggestion
# ACCESS-NRI Interactive Data Catalog
```

> The ACCESS-NRI Interactive Data Catalogue is a web-based application that allows users to explore and discover climate data products available on NCI's Gadi supercomputer.

Reviewer: ```suggestion
The ACCESS-NRI Interactive Data Catalogue is a web-based application that allows users to explore and discover climate data products available on NCI's _Gadi_ supercomputer.
```

It provides an intuitive interface for searching and filtering datasets based on various attributes such as variables, models, frequencies, and realms.

The Interactive Catalogue also allows users to subset and query data in order to select the minimal subset of data files they wish to open on gadi, and provides Python code snippets to allow users to load the selected data using the ACCESS-NRI Intake Catalogue.

It is not a separate catalogue from the ACCESS-NRI Intake Catalogue, but rather a complementary tool that provides a user-friendly way to interact with and share the same underlying data, without having to log into Gadi or write code.

# Explore the Interactive Data Catalog

> The Interactive Data Catalogue is hosted on GitHub Pages and can be accessed at the following URL:<br>

Reviewer: I would not add information on the hosting, as it would need to be changed in case the hosting changes.

```suggestion
The Interactive Data Catalogue can be accessed at the following URL:<br>
```
